content/en/docs/setup/additional-setup/compatibility-versions/index.md

```diff
@@ -40,9 +40,9 @@ $ helm install ... --set compatibilityVersion={{< istio_previous_version >}}
 
 ## When should I use compatibility versions?
 
-Compatibility versions are recommended to be used only when an incompatibility is found, rather than as the default.
-Each compatibility version will only persist for a few releases, so eventually you will need to migrate to the new behavior.
-Currently, each compatibility version is intended to remain for at least two versions, though this is subject to change.
+Compatibility versions should be used only when an incompatibility between releases exists, as a temporary measure. Eventually, you will need to migrate to the new behavior.
+
+Compatibility versions for a release will be removed and will no longer be supported when the release they refer to reaches end-of-life. Refer to the [https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases](current Istio release support status chart for the status of specific releases).
```

Comment:
```suggestion
Compatibility versions should be used only when an incompatibility between releases exists, as a temporary measure. You should plan to migrate to the new behavior as soon as practical.

Compatibility versions for a release will be removed, and will no longer be supported, when the release they refer to reaches end-of-life. Refer to the [current Istio release support status chart](https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases) for the status of specific releases.
```

Reply:
Made the language stronger, active, and fixed the markdown for your link.

Reply:
the one thing my brain refuses to remember is whether markdown links are []() or [](). it's very annoying.

Reply:
okay actually it refuses to remember a lot of things, but that's the one that annoys me the most.